docs/source/deployguide/cadeploy.md

```diff
@@ -0,0 +1,597 @@
+
+   **Important:** Make note of these `fabric-ca-client/tls-ca/admin/msp/signcerts/cert.pem` and `fabric-ca-client/tls-ca/admin/msp/keystore/` files. These are your TLS signcert and private key that you will need to reference in the CA configuration .yaml file in the following steps.
+
+   **Tip:** The name of the `/msp/keystore/` private key file will be a long string ending in `_sk`. For convenience, you can rename the file to something simpler such as `key.pem` so that it is easier to reference in the future.
+
+   The `--enrollment.profile tls` flag is specified because we are enrolling against the TLS CA. Use of this flag means that the enrollment is performed according to the `usage` and `expiry` settings of the TLS profile that is defined in the `signing` section of the configuration .yaml file.
+
+
+The following diagram is a conceptual summary of the steps you perform to create a TLS CA server and enroll the bootstrap identity using the Fabric CA client:
+
+![ca-tls-flow](./ca-tls-flow.png)
+
+### Register and enroll the root CA bootstrap identity with the TLS CA
+
+The TLS CA server was started with a bootstrap identity which has full admin privileges for the server. One of the key abilities of the admin is the ability to register new identities. Each node in the organization that transacts on the network needs to register with the TLS CA. Therefore, before we set up the root (organization) CA, we need to use the TLS CA to register and enroll the CA bootstrap identity to get its TLS certificate and private key.  The following command registers the organization CA bootstrap identity `rcaadmin` and `rcaadminpw` with the TLS CA.
+
+```
+./fabric-ca-client register -d --id.name rcaadmin --id.secret rcaadminpw -u https://my-machine.example.com:7054  --tls.certfiles ../tls/tls-ca-cert.pem
+```
+
+Next, we need to enroll the `rcaadmin` user to generate the TLS certificates for the identity. In this case, we use the `-M` flag on the enroll command to designate where the generated intermediate CA TLS certificates should be stored for the `rcaadmin` user. Because these certificates are for a different identity, it is a best practice to put them in their own folder. Therefore, instead of generating them in the default `msp` folder, we will put them in a new folder named `msp-rca`.
+
+```
+./fabric-ca-client enroll -d -u https://rcaadmin:rcaadminpw@my-machine.example.com:7054 --tls.certfiles ../tls/tls-ca-cert.pem -M msp-rca --csr.hosts 'host1,*.example.com'
+```
+
+The organization root CA TLS signed certificate is generated under `fabric-ca-client/tls-ca/admin/msp-rca/signcert` and the private key is available under `fabric-ca-client/tls-ca/admin/msp-rca/keystore`. When you deploy the organization CA you will need to point to the location of these two files in the `tls` section of the CA configuration .yaml file.
```

Comment:
Again, I'm not understanding why we'd want the rca credentials under the /admin directory.  Isn't the rca identity completely different from the admin identity?
See my comment on line 185.

Reply:
I am really glad you made this comment. I changed up the folder structure to be the following which I like much better:
```
fabric-ca-client
    └── tls-ca
      ├── admin
      ├── rcaadmin
      ├── icaadmin
      └── tls
```